### ISSUE TITLE
About generating image

### ISSUE THREAD

**Issue by LWei-gfkd** [grey-eye/talking-heads#29]
The generated image looks like a 4*4 image, and you can't see the original avatar at all.And after iterations hundreds of times, the loss value becomes nan.

**Comment by LWei-gfkd** [comment]
And after iterations hundreds of times, the loss value becomes nan.

**Comment by castelo-software** [comment]
@LWei-gfkd. As the readme explains, I haven't gotten around to implementing fine-tuning. For now only pre-training with the feed forward model has been finished.
If you want to help us implement it, feel free to update us on your findings here https://github.com/grey-eye/talking-heads/issues/26

If you just want to play with the feed forward model, change the `FEED_FORWARD = False` in the config file to True.
